INSTALL.md

```diff
@@ -36,7 +36,24 @@ Vagrant box will take a few minutes for downloading and installing all
 requirements and display a success message on console when done. Go to
 "Configure website" section below to continue.
 
-### Option 2: install manually on a server
+### Option 2: run locally with Docker
+
+Required software on your local machine:
+
+* [Docker](https://docs.docker.com/install/) 
+
+Procedure:
+
+Build the image and start the containers.
+
+* `docker-compose build`
+* `docker-compose up`
+
+Open http://localhost:8000/install.php and configure the site.
```

Comment:
Isn't this contradictory with the "expose 80" statement from Dockerfile? (note I don't know anything about Docker yet, so I'll test your change locally)

[edit] Ah it seems there is a port binding in docket-compose.yml. Please disregard, I'll definitely need to test anyway :)